src/services/VaultService/arbitrum.ts

```diff
@@ -17,33 +16,25 @@ import {
     QueryParam,
     DEFAULT_QUERY_PARAM,
     StrategyMetaData,
+    StrategyWithReports,
 } from '../../types';
 
 type StrategyBasicData = {
     [vault: string]: StrategyApi[];
 };
 
-type VaultGQLResult = {
-    id: string;
-    strategies: StrategyApi[];
-};
-
-type GQLResult = {
-    data: {
-        vaults: VaultGQLResult[];
-    };
-};
-
 export default class ArbitrumService implements VaultService {
```

Comment:
This file is modified for 2 reasons
1. remove all direct use of subgraph and instead use the new Subgraph class which exposes the same functionality
2. encapsulate the StrategyReport logic to pull strat reports via Subgraph. This previously occurred in a random util, but is now apart of the VaultService interface.